yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -627,8 +630,9 @@ export class Sequencer {
     this.log.debug('Creating block proposal for validators');
     const proposal = await this.validatorClient.createBlockProposal(block.header, block.archive.root, txHashes);
     if (!proposal) {
-      this.log.warn(`Failed to create block proposal, skipping collecting attestations`);
-      return undefined;
+      const msg = `Failed to create block proposal`;
+      this.log.error(msg);
```

Comment:
I believe this gets logged by the `catch` clause in `buildBlockAndAttemptToPublish`, no need to re-log here. But it doesn't harm either.

Reply:
Removed